packages/biometric-ed25519/src/fido2.ts

```diff
@@ -11,7 +11,7 @@ export class Fido2 {
             rpName,
             challengeSize: 128,
             attestation: 'none',
-            cryptoParams: [-8, -7, -257],
+            cryptoParams: [-7],
```

Comment:
Let's add a comment that we only support ECDSA signatures above this line. Since we're going against the recommended implementation guidelines that we should at least support -7 and -257